x-pack/plugins/rollup/public/crud_app/sections/job_list/job_list.js

```diff
@@ -141,9 +141,8 @@ export class JobListUi extends Component {
             <p>
               <FormattedMessage
                 id="xpack.rollupJobs.jobList.emptyPromptDescription"
-                defaultMessage={`
-                Rollup jobs summarize and store historical data in a smaller index for future analysis.
-                `}
+                defaultMessage="Rollup jobs summarize and store historical data in a smaller index
+                  for future analysis."
```

Comment:
**nit:** please align the text with `defaultMessage` or place the message in one line:
```
                defaultMessage="Rollup jobs summarize and store historical data in a smaller index
                for future analysis."
```
or
```
                defaultMessage="Rollup jobs summarize and store historical data in a smaller index for future analysis."
```

Reply:
I would like to leave the text indented whenever the text wraps, to visually differentiate a wrapping value from multi-line props.